### DISCUSSION TITLE
What does tar_make() do when it skips a target?

### DISCUSSION THREAD

**Discussion by potash** [ropensci/targets#1009] [Help]
I see that `tar_make()` prints every target that it skips. For my current project with hundreds of targets this takes a couple of seconds. I am wondering whether this delay is purely from my terminal buffering-- in which case, can I skip printing the skips?-- or is targets actually doing something for each of these skips? Thanks!

**Comment by wlandau** [comment]
A bunch of that time is indeed spent printing messages (usually). You might try tar_make(reporter = verbose_positives).

Checking files, time stamps, and hashes might take a while too, depending on the speed of your file system.
